battalion.sh

```diff
@@ -94,6 +95,9 @@ do
         --nmap)
             NMAP_ENABLED=true
             ;;
+        --nmap-aggressive)
```

Comment:
This is fine for now. Eventually we should consider doing something like this:

```
--nmap [light|medium|aggressive]
```

Or something to that effect if we add more control over our scan.